_includes/docs/pe/user-guide/rbac.md

```diff
@@ -1,77 +1,94 @@
 * TOC
 {:toc}
 
+## Overview
 
-**Below you can find all the information and instructions about this feature**, or you can see the video tutorial for step-by-step instructions on how to use it.
+Identity access and management (IAM) is essential for today’s business security strategy. 
+Many IAM systems use a method known as role-based access control (RBAC) to assign permissions for who can do what within specified IT resources.
 
-<br/>
-<div id="video">  
-    <div id="video_wrapper">
-        <iframe src="https://www.youtube.com/embed/xpnYzSiDiJo" frameborder="0" allowfullscreen></iframe>
-    </div>
-</div> 
+Role-Based Access Control (RBAC) allows to create and grant advanced access by assigning a set of permissions. 
+RBAC roles refer to the levels of access that users may have. Access to resources can be limited to specific operations, such as viewing, creating, writing, or deleting data. 
+Similarly, you can restrict access to sensitive information, increasing business security.
 
 ## ThingsBoard CE vs PE security features comparison
 
-ThingsBoard Community Edition (TB CE) supports a straight-forward security model with three main roles: System administrator, Tenant administrator, and Customer user. 
+#### ThingsBoard Community Edition
+
+![image](/images/user-guide/security/rbac/TB_CE.png)
+
+ThingsBoard Community Edition supports a straight-forward security model with three main roles: *System administrator*, *Tenant administrator*, and *Customer user*. 
 A system administrator is able to manage tenants, while a tenant administrator manages devices, dashboards, customers, and other entities that belong to a particular tenant.
 Customer user is able to view dashboards and control devices that are assigned to a specific customer.
-TB CE functionality is sufficient for a lot of simple use cases, especially building real-time [end-user dashboards](/docs/{{docsPrefix}}user-guide/dashboards/).
- 
-ThingsBoard Professional Edition (TB PE) brings much more flexibility in terms of user, customer, and role management. 
+ThingsBoard Community Edition functionality is sufficient for a lot of simple use cases, especially building real-time [end-user dashboards](/docs/{{docsPrefix}}user-guide/dashboards/).
+
+#### ThingsBoard Professional Edition
+
+![image](/images/user-guide/security/rbac/TB_PE.png)
+
+ThingsBoard Professional Edition brings much more flexibility in terms of user, customer, and role management. 
 It is designed to cover use cases for businesses and enterprises with multiple user groups that have different permissions but may interact with the same devices and assets. 
 
-TB PE security model was significantly improved in v2.3 to enable new security features and to support advanced RBAC for IoT applications. For example:
+ThingsBoard supports the "recursive" customer hierarchy with an unlimited number of sub-customers.
+The root-level Owner is Tenant. Each Owner may have multiple Entity Groups, User Groups, and Customer Groups.
```

Comment:
Please define the Owner. Something like "Each entity (device, asset, dashboard, user, etc) has an Owner. This block requires improvement.